cluster/charts/rook/values.yaml.tmpl

```diff
@@ -32,6 +32,10 @@ logLevel: INFO
 ##
 rbacEnable: true
 
+## If true, create & use PSP resources
+##
+pspEnable: true
```

Comment:
Can PSP resources cause issues in clusters without PodSecurityPolicies (admission controller) activated or is PSP enabled by default?

Reply:
no.. its not going to cause issues.. It will create the PSPs but it won't enforce if PSP admission controller is not enabled.

Reply:
I believe all the clusters in Jenkins CI doesn't have PSP, but it worked!